articles/active-directory/develop/quickstarts/active-directory-javascriptspa.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
General comment...I really like we are doing a simple sample of get token and then get some resources.  It would be nice to see a "composed" sample with all the pieces put together.  Did I miss a link here where you just download a single file and run it?  Just like the old quickstarts from the landing page.
https://identity.microsoft.com/Docs/Web

Reply:
Yes, Option 2, Step 2 [here](https://review.docs.microsoft.com/en-us/aad-quickstarts-test/articles/active-directory/develop/quickstarts/active-directory-javascriptspa)?